src/server/cluster/cluster_family.cc

```diff
@@ -526,7 +524,7 @@ void ClusterFamily::DflyClusterConfig(CmdArgList args, ConnectionContext* cntx)
   DCHECK(tl_cluster_config != nullptr);
 
   // TODO rewrite with outgoing migrations
-  if (!StartSlotMigrations(new_config->GetIncomingMigrations(), cntx)) {
+  if (!StartSlotMigrations(new_config->GetOutgoingMigrations(), cntx)) {
```

Comment:
what happends if the client resends the same config? from what I understand from this flow you will start the migration again?

Reply:
yes, I have created a separate ticket for corner cases.